proto/minder/v1/minder.proto

```diff
@@ -966,14 +966,6 @@ message GitHubProviderConfig {
 message Provider {
     string name = 1;
 
-    // Context defines the context in which a provider is evaluated.
-    // Given thta a provider is a top level entity, it may only be scoped to
-    // an organization.
-    message Context {
-        string organization = 1;
-        string project = 2;
-    }
-
```

Comment:
I totally glossed over this, thinking it was unused (despite the fact that it's in the next unchanged diff line).

This is an unsafe change, since proto's wire format depends only on tag numbers.  (It will compile, but it's not compatible with old client / new server).  The `Context` below was referencing `minder.v1.Provider.Context`, which was defined as:

```proto
message X {
  string organization = 1;
  string project = 2;
}
```

It was replaced with the `minder.v1.Context` message, which is defined as:

```proto
message Y {
    optional string provider = 1;
    optional string project = 3;
}
```

While the _types_ of the two don't conflict, message `X` believes that tag 1 is an organization, while `Y` believes that tag 1 is a provider.  This means that a client might send a message that says "create a Provider in organization `f31eag...` and project `0a31fd...`", but the server interprets this as "create a Provider with provider name `f31eag...` and retired organization `0a31fd...` (and project "")".